### DISCUSSION TITLE
Tab can get the ID of the current activation page

### DISCUSSION THREAD

**Discussion by kimdiego2098** [dotnetcore/BootstrapBlazor#3350] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the solution you'd like

Tab can get the ID of the current activation page

The idea is to display the current page in full screen, so an Id or Element is required

**Comment by bb-auto** [comment]
@kimdiego2098 Thank you for contacting us. We will give feedback later.

**Comment by ArgoZhang** [comment]
@kimdiego2098 `TabItem Id` 你拿到是没用的啊。你全屏用自己的根节点啊。跟 `Tab` 没有关系啊

**Comment by ArgoZhang** [comment, marked as answer]
@kimdiego2098 TabItem 增加一个参数 `ShowFullScreen` 这样 TabItem 上自动出现全屏按钮
